src/main/main.ts

```diff
@@ -9,10 +9,15 @@
 debug({
 	showDevTools: false
 });
-let mainWindow: BrowserWindow | null;
+let mainWindow: BrowserWindow | null;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5693151)